docs/fees_and_rewards/Dynamic_Issuance.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Dynamic Issuance
+sidebar_position: 3
+description: Dynamic Issuance Formulas
+keywords:
+    - fees
+    - rewards
+    - WIP
+last_update:
+  date: 04/16/2024
+  author: Saeid Yazdinejad
+---
+import Collapsible from '@site/src/components/Collapsible/Collapsible';
+
+# Dynamic Issuance
+As opposed to having a fixed issuance rate, the Subspace Network implements a dynamic issuance rate based on network usage, where, roughly, the inflation rate is inversely proportional to the utilization rate of the blockspace. 
+
+TLDR: The farmer who proposed a block gets some fresh SSC + fees, and voters get some fresh SSC regardless of what the proposer got.
+
+
+[Dynamic Issuance Specification](https://www.notion.so/Dynamic-Issuance-Specification-3bdf63955cb943489347889775d71c24?pvs=21)
+
+
+# Space Race
+
+The rewards issuance is not enabled by default at the chain genesis. After the genesis block, several things need to happen before authoring blocks by anyone, and  [rewards can be enabled](https://subnomicon.subspace.network/docs/consensus/genesis). Then:
+
+1. Define a target space pledged to the network at which the rewards will be enabled (i.e., 8PiB for Gemini-3h). In the runtime, it is defined in terms of the target value of the solution range [`solution_range_for_rewards`](https://github.com/subspace/protocol-specs/issues/13).
+2. Identify an assumed current space pledged `initial_solution_range`.
+3. Enable solution range adjustment starting at `initial_solution_range` at the next era 
+4. Set in the runtime the `solution_range_for_rewards` below which to automatically enable rewards.
+5. Enable block authoring by anyone.
+6. Farmers will start producing blocks and the solution range will be updated every era as defined in [Solution Range Updates](https://www.notion.so/Solution-Range-Updates-a7205a46a20c42e5838d0bc332c51bed?pvs=21) 
+7. Eventually, as farmers pledge more space, the network’s solution range will reach a value ≤ `solution_range_for_rewards` at a certain block $B$.
+8. The `RewardsEnabled` runtime item is set to `true`. The rewards will start to be paid to voters and proposer of the very next block.
+
+
+
+# Issuance Function
+
+The issuance function currently consists of two components: a decaying component and a utilization-based component. The decaying component issues a constant reward for some initial interval and gradually reduces it over time. The utilization-based component issues a reward based on the average utilization of blockspace in recent blocks.
+Such construction allows for more decaying components to be included for additional incentivization.
+
+**Full  Block**
```

Comment:
Please make all of these section titles ### instead of bold face. This way it will be possible to directly link to specific sections from other places.